components/eamxx/src/physics/rrtmgp/eamxx_rrtmgp_process_interface.cpp

```diff
@@ -105,9 +108,13 @@ void RRTMGPRadiation::set_grids(const std::shared_ptr<const GridsManager> grids_
   // Set of required gas concentration fields
   for (auto& it : m_gas_names) {
     // Add gas VOLUME mixing ratios (moles of gas / moles of air; what actually gets input to RRTMGP)
-    if (it == "o3") {
-      // o3 is read from file, or computed by chemistry
-      add_field<Required>(it + "_volume_mix_ratio", scalar3d_layout_mid, molmol, grid_name);
+    if (it == "o3" && !m_transient_ozone) {
+      if (!m_transient_ozone) {
```

Comment:
We should probably call this `m_prescribed_ozone` instead of transient, and then always require this input is present until we have ozone computed by chemistry. I want to completely abandon the practice of sticking it in the initial condition file and remove it from the ICs.

Reply:
Actually, I think I misread this...